packages/replicache/src/dag/lazy-store.test.ts

```diff
@@ -547,15 +545,15 @@ test('cache eviction suspension, basic test of evict on write', async () => {
       // would evict testValue1Chunk, but doesn't because evicts are suspended
       await write.getChunk(testValue3Chunk.hash);
     });
-    expect(lazyStore.getCachedSourceChunksSnapshot()).to.deep.members([
+    expect(lazyStore.getCachedSourceChunksSnapshot()).toEqual([
```

Comment:
I'm not sure about this transformation.

`to.deep.member` is unordered if I'm not mistaken?